modules/op-mirroring-images-to-run-pipelines-in-restricted-environment.adoc

```diff
@@ -109,14 +109,14 @@ Namespace:		openshift
 +
 [source,terminal]
 ----
-$ oc image mirror registry.redhat.io/ubi8/go-toolset:1.14.7 <mirror-registry>:<port>/ubi8/go-toolset
+$ oc image mirror registry.redhat.io/ubi9/go-toolset:1.14.7 <mirror-registry>:<port>/ubi9/go-toolset
```

Comment:
Seems there is no 1.14.7 tag for go-toolset on Ubi9

suggest to change to : 
`oc image mirror registry.redhat.io/ubi9/go-toolset:latest <mirror-registry>:<port>/ubi9/go-toolset`